library/psa_crypto.c

```diff
@@ -2514,7 +2517,7 @@ static psa_status_t psa_hmac_setup_internal( psa_hmac_internal_data *hmac,
 }
 #endif /* MBEDTLS_PSA_BUILTIN_ALG_HMAC */
 
-static psa_status_t psa_mac_setup( psa_mac_operation_t *operation,
+static psa_status_t psa_mac_setup( mbedtls_psa_mac_operation_t *operation,
```

Comment:
Was this an FYI or do you expect me to change the history with a rebase again?

Reply:
Currently the commit "Move the MAC operation structure into the driver headers" does not compile. If you don't mind amending the commit I'd prefer. This would change the history in a easy to track way (force-push diff) from my point of view.